README.rst

```diff
@@ -40,6 +40,48 @@ It returns a list of values that correspond to the limits of the classes (starti
     # 1st class              1st class            2nd class         3rd class          4th class         5th class
     # (Minimum value)                                                                                    (Maximum value)
 
+
+
+This package also support a JenksNaturalBreaks class as interface (inspired by scikit-learn classes). The fit and group behavior is slightly different with jenks_breaks. To retain size, fit and group will use only the inner bound (without min and max bound). All value below the min bound will be included in the first group and all value higher than the max bound will be included in the last group.
+
+
+
+.. code:: python
+
+    >>> from jenkspy import JenksNaturalBreaks
+
+    >>> x = [0,1,2,3,4,5,6,7,8,9,10,11]
+
+    >>> jnb = JenksNaturalBreaks
```

Comment:
Should be `JenksNaturalBreaks()` ?

Reply:
Done. All fixed and ready to be merged, I think.

*still a noob in PR world